frontend/src/component/feature/FeatureView/FeatureOverview/FeatureOverviewMetaData/FeatureOverviewMetaData.tsx

```diff
@@ -97,55 +79,46 @@ const FeatureOverviewMetaData = () => {
 
     const showDependentFeatures = useShowDependentFeatures(project);
 
-    const FlagTypeIcon = getFeatureTypeIcons(type);
-
     return (
         <>
             <StyledMetaDataContainer>
-                <StyledMetaDataHeader data-loading>
-                    <FlagTypeIcon />
-                    <h2>{capitalize(type || '')} flag</h2>
-                </StyledMetaDataHeader>
-                <ConditionallyRender
-                    condition={Boolean(description)}
-                    show={
+                <div>
+                    <StyledTitle>Flag details</StyledTitle>
+                    {description ? (
                         <StyledMetaDataItem data-loading>
                             <StyledMetaDataItemText>
                                 {description}
                             </StyledMetaDataItemText>
                         </StyledMetaDataItem>
-                    }
-                />
+                    ) : null}
+                </div>
                 <StyledBody>
                     <StyledMetaDataItem>
                         <StyledMetaDataItemLabel>
-                            Project:
+                            Flag type:
                         </StyledMetaDataItemLabel>
                         <StyledMetaDataItemText data-loading>
-                            {project}
+                            {capitalizeFirst(type || '')} flag
```

Comment:
This can also be done by CSS `::first-letter: { text-transform: uppercase }` which will also capitalize "flag" if there is no type 💁🏼 Not important, but this feels more like a CSS thing to me 🤷🏼

Reply:
I'm using what we already have, plus I did not have good time debugging and finding what's causing transform with `::` pseudo-classes in the past.